react/features/toolbox/components/web/ToolbarButton.js

```diff
@@ -53,8 +88,12 @@ class ToolbarButton extends AbstractToolbarButton<Props> {
         return (
             <div
                 aria-label = { this.props.accessibilityLabel }
+                aria-pressed = { this.props.toggled }
                 className = 'toolbox-button'
-                onClick = { this.props.onClick }>
+                onClick = { this.props.onClick }
+                onKeyDown = { this._onKeyDown }
+                role = 'button'
+                tabIndex = { 0 }>
```

Comment:
ditto

Reply:
same as above for this div, without it would not be reachable via keyboard